test/test_status.py

```diff
@@ -32,3 +36,49 @@ def test_status(dirtyrepo):
     for filepath, status in git_status.items():
         assert filepath in git_status
         assert status == git_status[filepath]
+
+
+def test_status_untracked_no(dirtyrepo):
+    git_status = dirtyrepo.status(untracked_files="no")
+    not any(status & pygit2.GIT_STATUS_WT_NEW for status in git_status.values())
+
+
+@pytest.mark.parametrize(
```

Comment:
These tests don't actually cover all of the possible scenarios, such as having tracked subdirectories with ignored files, tracked subdirectories with untracked files, untracked subdirectories with ignored files, etc, etc. Might improve these tests soon.

Reply:
We're just passing the options to libgit2, the purpose of the test suite is not to test libgit2, which already has its own test suite. So better to keep the tests simple, it's good the way they are already.